openwisp_radius/tests/test_api/test_rest_token.py

```diff
@@ -90,6 +90,20 @@ def test_user_auth_token_400_organization(self):
         self.assertEqual(r.status_code, 400)
         self.assertIn('is not member', r.json()['non_field_errors'][0])
```

Comment:
A user will be able to register to a new organization only if it is already a member of an existing organization.